com.unity.shadergraph/Tests/HeadlessPreview/PreviewTests.cs

```diff
@@ -526,7 +526,7 @@ public void NodePreview_GetFuncAndBlockShaderCode()
             previewMgr.RequestNodePreviewShaderCodeStrings("PassNodeInst", out _, out _, out var blockCode, out var funcCode);
 
             // this block code is generated by the interpreter and is welcome to be modified if expectations of the interpreter change.
-            string expectedBlock = "void PassThrough(out float1 Out, float1 In)\n{\nOut = In.x;\n}\nShaderGraphBlockBlock::ShaderGraphBlockOutput SYNTAX_PassNodeInstMain(ShaderGraphBlockBlock::ShaderGraphBlockInput In)\n{\nfloat1 SYNTAX_PassNodeInst_Out = float1(0);\nPassThrough(SYNTAX_PassNodeInst_Out, float1(1));\nShaderGraphBlockBlock::ShaderGraphBlockOutput output;\noutput.BaseColor = float3(SYNTAX_PassNodeInst_Out, SYNTAX_PassNodeInst_Out, SYNTAX_PassNodeInst_Out);\nreturn output;\n}\n";
+            string expectedBlock = "void PassThrough(out float1 Out, float1 In)\n{\nOut = In.x;\n}\nShaderGraphBlock_PassNodeInstBlock::ShaderGraphBlock_PassNodeInstOutput SYNTAX_PassNodeInstMain(ShaderGraphBlock_PassNodeInstBlock::ShaderGraphBlock_PassNodeInstInput In)\n{\nfloat1 SYNTAX_PassNodeInst_Out = float1(0);\nPassThrough(SYNTAX_PassNodeInst_Out, float1(1));\nShaderGraphBlock_PassNodeInstBlock::ShaderGraphBlock_PassNodeInstOutput output;\noutput.Out = SYNTAX_PassNodeInst_Out;\nreturn output;\n}\nShaderGraphBlock_PassNodeInst_REMAPBlock::ShaderGraphBlock_PassNodeInst_REMAPOutput SYNTAX_ShaderGraphBlock_PassNodeInst_REMAPMain(ShaderGraphBlock_PassNodeInst_REMAPBlock::ShaderGraphBlock_PassNodeInst_REMAPInput inputs)\n{\nShaderGraphBlock_PassNodeInst_REMAPBlock::ShaderGraphBlock_PassNodeInst_REMAPOutput output;\noutput.BaseColor = float3(inputs.Out, inputs.Out, inputs.Out);\nreturn output;\n}\n";
```

Comment:
Updating expected code to take into account block remap

Reply:
Does this pass?

Reply:
My bad- I had the wrong version of the branch.